README.md

```diff
@@ -1,8 +1,50 @@
 # ansible-dkim
 
-Ansible role for configuring [Postfix](http://www.postfix.org/) with [OpenDKIM](http://opendkim.org/) support on [Ubuntu](https://ubuntu.com/).
+Ansible role for configuring [Postfix](http://www.postfix.org/) with [OpenDKIM](http://opendkim.org/). Works on [Debian](https://debian.org) distributions and derived like [Ubuntu](https://ubuntu.com/).
 
-### Example playbook
+## Description
+
+This role confiures DKIM mail singing service in a hosts that works as e.ail MTA. 
+
+The role: 
+* installs and configures opendkim,
+* creates private and public dkim keys for the domains it has to sign, declared in the `dkim_domains` variable,
+* installs postfix and configures it to pass all the messages of the configured domains to be signed by OpenDKIM,
+* shows the DNS records with the public keys that must be defined in the public DNS of the domains we sign.
+
+## Requirements
+
+The role requires that you configure all the rest of the mail management and you will need to have access to the DNS configuration of the domains you are requiesting to sign. At the end, the role will give you the DNS records with the public keys of the domain that you will have to publish in the global DNS system.  
+
+## Role variable
+
+### Opendkim package parameters
+
+|  Variable     |   Default value   |   Description  |
+|:-------------------:|:------------------------:|:------------:|
+| `dkim_default_config_file:` | /etc/default/opendkim | Opendkim Default values configuration file |
+| `dkim_opendkim_config:` | /etc/opendkim | Opendkim configuration folder |
```

Comment:
Shouldn't be `dkim_opendkim_config_dir` since it's directory?

Reply:
Ok. I replace. I always doubt between short and meaningful. For instance the previous variable `dkim_default_config_file` should normally mention opendkim package.